golpa.go

```diff
@@ -404,3 +407,45 @@ func (model *Model) SolveWithContext(ctx context.Context) (res *SolveResult, err
 
 	return ret, err
 }
+
+// WriteLPFile dumps the model to the specified file in lp format.
+func (model *Model) WriteLPFile(filename string) error {
+	model.mu.RLock()
+	defer model.mu.RUnlock()
+
+	ret := C.write_lp(model.prob, C.CString(filename))
+
+	if ret != C.TRUE {
+		return fmt.Errorf("model not written successfully")
+	}
+
+	return nil
+}
```

Comment:
I don't think we need to increase the API surface with two "dump" methods. The second one (returning the content) is flexible enough and feels a lot more idiomatic, so I think we can get rid of this one.

Reply:
Ok, I removed this one.